src/libPMacc/include/algorithms/math/floatMath/trigo.tpp

```diff
@@ -83,7 +84,7 @@ struct Sinc<float>
 {
     typedef float result;
 
-    HDINLINE double operator( )(const float& value )
```

Comment:
Please, confirm that this was indeed a bug and not some functionality hidden in some weird way.